android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -552,6 +552,25 @@ public void releaseViews()
 	 */
 	public abstract TiUIView createView(Activity activity);
 
+	
+	/**
+	 * Adds multiple child to this view proxy.
+	 * @param childs The child view proxies to add.
+	 * @module.api
+	 */
+	@Kroll.method
+	public void add(TiViewProxy[] childs)
```

Comment:
You mean all the checks? I thought it would be a good idea to leave them where they are, since the method is called for every child anyway. What else do I need to do? It seems that it still does not work.